release notes/v1.0.0.md

```diff
@@ -0,0 +1,127 @@
+k6 `v1.0.0` is here 🎉!
+
+After 9 years, 9k+ commits and 58 releases from the first commit in 2015, we're excited to announce that we're going to leave behind the `v0.x` series. Starting from now k6 aligns with the industry standard of following the semantic versioning scheme and will release this and future versions by following its new stability policy.
+
+This release includes:
+
+- Native support for extensions in the Cloud
+- New test failure control with `execution.test.fail`
+- Documentation for the stability guarantees
+
+## New features
+
+### Native support for extensions in the Cloud [#4670](https://github.com/grafana/k6/pull/4671)
+
+The new Binary provisioning feature automatically requests and uses custom k6 binaries with the required extensions for your tests. This allows you to run scripts that use extensions without manually rebuilding k6 as it was in the past by depending on tools like `xk6`. The system caches binaries locally for efficiency, and any additional runs with the same dependencies will use the same binary and will run faster.
+
+Binary provisioning is available for all k6 Cloud users (free and paid plans), and it's enabled by opt-in with the feature flag `K6_BINARY_PROVISIONING=true`.
```

Comment:
Might be worth detailing or linking to a doc on how to set this env var up. I presume it's something the customer can do in their own GCk6 instance?

Reply:
This is the doc we currently have for the feature: https://grafana.com/docs/grafana-cloud/testing/k6/author-run/use-k6-extensions/.

I need to update it to include the feature flag.

Reply:
I see @pablochacin made a docs PR [here](https://github.com/grafana/k6-docs/pull/1937/files), I just missed reviewing it. :( 

I'm going to try to do that first thing tomorrow. 🙇